databaseQueries/programUsers.js

```diff
@@ -0,0 +1,45 @@
+module.exports= class Programs{
+    static findProgramJoined(programId,userId) {
```

Comment:
@ankitpws can we make change the function name to programUsersDocument, and generalise the function